dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -441,7 +441,7 @@ private boolean hasArgumentConfigProps(Map<String, String> configProperties, Str
     }
 
     protected MethodConfig getMethodByName(String name) {
-        if (methods != null && methods.size() > 0) {
+        if (methods != null && !methods.isEmpty()) {
```

Comment:
**Optimization Issue**: Optimization by replacing 'methods.size() > 0' with '!methods.isEmpty()' to enhance code readability and maintainability. <br> **Fix**: Adopt the isEmpty method for checking whether the collection is not empty, improving code readability. <br> **Code Suggestion**: 
 ```
 -        if (methods != null && methods.size() > 0) {
 +        if (methods != null && !methods.isEmpty()) {
 ```